scientist-softserv/harvard-curiosity
# Security bumps and OAI

**Security bumps and OAI**
* * *


# What does this Pull Request do?
2 things:
1. Resolves some security alerts for packages in gem file and node
2. adds the ability to put an optional solr query in the solr harvest

# How should this be tested?

For 1. make sure the site comes up and works

For 2. you'll want to do a solr harvest. Either create a junk collection to test with or use the courses collection and do the following:

- Go to the Item Harvest screen in your collection and select Solr Harvest. You'll see that there's a new field for solr query. If doing acats-courses, in that field put the following filter:  startYear:2005
- this will harvest 5480 items to your collection (will take a little while)
- also do a test with no filter, USE acats-calendars FOR THIS (smallest collection)

I've done both of these on curiosity-dev as well.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n
(but maybe we want some? not sure if we could make an integration test for this)

# Interested parties
Tag (@ mention) interested parties
